ol-fyi-local-infra/compose.yml

```diff
@@ -32,7 +32,7 @@ services:
     volumes:
       - ./volumes/postgres/var/lib/postgresql/data:/var/lib/postgresql/data
     ports:
-      - 5432:5432
+      - 5434:5432
```

Comment:
Couldn't immediately see what's behind this change. Was the port just wrong before?

Reply:
Local collision on the port mapping.

Is this docker compose being used in prod?

Reply:
It isn't but I wondered why it changed in this PR. Seemed to be no reason. Is it so it runs on your machine?